### DISCUSSION TITLE
Some translations that may not be very accurate

### DISCUSSION THREAD

**Discussion by sissi815zx** [zsviczian/obsidian-excalidraw-plugin#246] [General]
I'm not very professional, these are translations that may not be very accurate
[obsidian Excalidraw translate.txt](https://github.com/zsviczian/obsidian-excalidraw-plugin/files/7527445/obsidian.Excalidraw.translate.txt)

Display 
显示

New drawing to match Obsidian theme与
新建绘图沿用obsidian明暗主题

If theme is dark, new drawing will be created in dark mode. This does not apply when you use a template for new drawings. Also this will not effect when you open an existing drawing. Those will follow the theme of the template/drawing respectively.
如果主题为暗，则将在暗模式下创建新图形。当您为新图形使用模板时，这不适用。此外，打开现有图形时，此操作也不会生效。这些将分别遵循模板/图纸的主题。

Existing drawings to match Obsidian theme
让已存在的绘图与obsidian明暗主题相匹配

If theme is dark, drawings will be opened in dark mode. If your theme is light, they will be opened in light mode
如果主题为暗，则图形将以暗模式打开。如果你的主题是轻，它们将在明亮模式下打开

Zoom to fit on view resize
缩放以适应视图调整大小

Zoom to fit drawing when the pane is resized
调整窗格大小时缩放以适合图形

Zoom to fit max ZOOM level
缩放以适应最大缩放级别

Set the maximum level to which zoom to fit will enlarge the drawing. Minimum is 0.5 (50%) and maximum is 10 (1000%).
设置“缩放到合适”将放大图形的最大级别。最小值为0.5（50%），最大值为10（1000%）。

Open in adjacent pane
在相邻标签栏中打开

When CTRL/CMD+SHIFT clicking a link in Excalidraw by default the plugin will open the link in a new pane. Turning this setting on, Excalidraw will first look for an existing adjacent pane, and try to open the link there. Excalidraw will first look too the right, then to the left, then down, then up. If no pane is found, Excalidraw will open a new pane
默认情况下，当CTRL/CMD+SHIFT单击Excalidraw中的链接时，插件将在新标签栏中打开该链接。打开此设置后，Excalidraw将首先查找现有的相邻标签栏，并尝试打开其中的链接。Excalidraw会先向右找，然后向左找，然后向下找，然后向上找，如果未找到标签栏，Excalidraw将打开一个新窗格。

URL prefix
URL前缀

In PREVIEW mode, if the Text Element contains a URL link, precede the text with these characters. You can override this setting for a specific drawing by adding 'excalidraw-url-prefix: "*** "' to the file's frontmatter.
在预览模式下，如果文本元素包含一个URL链接，则在文本前面加上这些字符。你可以通过添加 " excalidraw-url-prefix:"****" "到文件的frontmatter中来覆盖特定绘图的这个设置。

Overflow wrap behavior of transcluded text
溢出包装行为的反括号文本

! [[doc#Aref1] (number} Number specifies the character count where the text should be wrapped. Set the text wrapping behavior of transcluded text. Turn this ON to force-wrap text (i.e. no overflow), or OFF to soft-wrap text (at the nearest whitespace).
! [[doc#Aref1] (number) number指定文本应被包裹的字符数。设置转载文字的包覆行为。打开它可以强制包裹文本（即没有溢出），或者关闭它可以软包裹文本（在最近的空白处）。

Page transclusion max char count
页面转换的最大字符数

The maximum number of characters to display from the page when transcluding an entire page with the ![[markdown pagel] format.
使用 ![[markdown pagel] 格式转换整个页面时从页面显示的最大字符数。

Use iframely to resolve page title 
使用iframely获得页面标题 

Use the  http://iframely.server.crestify.com/iframely? url= to get title of page when dropping a link into Excalidraw
当把一个链接放到Excalidraw中时，使用http://iframely.server.crestify.com/iframely? url=  来获得页面的标题。

Type of file to insert into the document
入文档的文件类型

When you embed an image into a document using the command palette this setting will specify if Excalidraw should embed the original excalidraw file or a PNG or an SVG copy. You need to enable auto-export PNG/ SVG (see below under Export Settings) for those image types to be available in the dropdown. For drawings that do not have a a correspondign PNG or SVG readily available the command palette action will insert a broken link. You need to open the original drawing and initiate export manually. This option will not autogenerate PNG/SVG files, but will simply reference the already existing files
当您使用 command palette 将图像嵌入文档时，此设置将指定Excalidraw嵌入的是原始的excalidraw文件，是PNG副本，还是SVG副本。您需要启用 auto-export PNG/SVG 自动导出PNG/SVG（ 见下面的导出设置），这些图像类型才能在下拉菜单中可用。对于没有对应的PNG或SVG的图纸，command palette 动作将插入一个broken link。你需要打开原始图纸并手动导出。这个选项不会自动生成PNG/SVG文件，而只是引用已经存在的文件。

(Here is a problem: For drawings that do not have "a" a correspondign PNG)

**Comment by zsviczian** [comment]
great! can I ask you you do a pull request on the language file and propose these changes?
https://github.com/zsviczian/obsidian-excalidraw-plugin/blob/master/src/lang/locale/zh-cn.ts
https://github.com/zsviczian/obsidian-excalidraw-plugin/blob/master/src/lang/locale/en.ts
